mmv1/products/compute/Image.yaml

```diff
@@ -225,6 +225,7 @@ properties:
           An optional SHA1 checksum of the disk image before unpackaging.
           This is provided by the client when the disk image is created.
         api_name: sha1Checksum
+        diff_suppress_func: 'tpgresource.Base64DiffSuppress'
```

Comment:
Is this newly necessary?

Reply:
Hi @SirGitsalot,

We had an interesting side-effect with [base64-encoded strings [1]](https://github.com/GoogleCloudPlatform/magic-modules/pull/12369#pullrequestreview-2540385326). For that reason my colleague @yeoldegrove added [this function [2]](https://github.com/GoogleCloudPlatform/magic-modules/pull/12369/commits/7e02c79f194d0eb253c1edac21460b7e63e78ade).

[1] https://github.com/GoogleCloudPlatform/magic-modules/pull/12369#pullrequestreview-2540385326
[2] https://github.com/GoogleCloudPlatform/magic-modules/pull/12369/commits/7e02c79f194d0eb253c1edac21460b7e63e78ade